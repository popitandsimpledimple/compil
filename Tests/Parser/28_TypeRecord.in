var point: record
end;
begin end.