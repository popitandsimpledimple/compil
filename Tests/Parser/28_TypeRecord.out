program
   decls
      vars
         var
            point
               record
               params
            no expression
   block
      compounds statements
